_component_design/radio-buttons.md

```diff
@@ -48,17 +38,23 @@ helix-ui-javascript: false
 
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}  
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}  
+
+## Best practices
 
-### Best practices
+Consider the following best practices when you construct radios:
 
-- Always use a label above the available options.
-- Keep the number of options to between two and five choices.
-- There is always a default option selected (preferably at the top or the list).
-- If the user clicks on another option, the first is unselected.
-- Radio buttons should only be used when the list can be vertical. If space is not available, consider a dropdown menu.
-- List options in an order that makes logical sense. Depending on the context, the order could be alphabetical or from most common to least common. For example, if listing accounts by name, list them alphabetically. In billing, paying the amount due should be listed above letting the user enter a custom payment amount.
-- Use checkboxes and radio buttons only to change settings, not as action buttons that make something happen.
+- By default, the radio at the top of the list is selected. The user can then make a different selection, if required.
+- When the user clicks a radio, the previously selected button is deselected.
+- Keep the number of options to two and five choices. If you’re providing more than five options, consider using a drop-down list.
+- List radio options vertically, with one choice per line. If space is not available, consider using a drop-down list.
```

Comment:
1) it's not called a "drop-down list". Currently it's called a "drop-down". 
so ftm, we should probably just cut the word "list"

2) should we also recommend they try a "single select strip"?

3) Should we add reference links to those pages here?

Reply:
Something like "Consider using a single select strip when you want text input fields on a form to dynamically change based on the user's selection."? And then I'll link to the single select strip page...

Reply:
"List radio options vertically, with one choice per line. If space is not available, consider using a drop-down selector or single select strip."

Then link to those two pages.
The info about changing the text inputs below is only sometimes accurate and can be handled in a couple of ways. That IS how payment method works, but it's not always that way...i figure we should just cut that info and revisit the idea in a Forms page that details how to build forms correctly with helix inputs.